examples/bitbucket/pipeline.yaml

```diff
@@ -0,0 +1 @@
+../example-pipeline.yaml
```

Comment:
If we use a new pipeline just for this with its own name instead of sharing one common pipeline, its a lot easier to find out which EL created the pipelinerun/taskrun i.e in the readme, we can say    `kubectl get pipelinerun| grep bitbucker-pr` or something